src/lib.rs

```diff
@@ -74,21 +74,28 @@ extern crate error_chain;
 #[macro_use]
 extern crate serde_derive;
 extern crate serde;
-#[macro_use] 
+#[macro_use]
 extern crate serde_json;
+#[macro_use]
+extern crate log;
 
 extern crate handlebars;
 extern crate pulldown_cmark;
 extern crate regex;
 
+#[cfg(test)]
 #[macro_use]
-extern crate log;
+extern crate pretty_assertions;
+#[cfg(test)]
+extern crate tempdir;
```

Comment:
Ah okay, what would you prefer me to do here? Do you think I should try to remove the `tempdir` dependency, rearrange the imports, or even move the `extern crate` statement to `loader/book.rs`?